UPGRADING.md

```diff
@@ -0,0 +1,56 @@
+UPGRADING
+--------
+
+This file documents how to upgrade an existing BHIMA installation to a new release.  To learn about the releases process, see [RELEASES](./RELEASES.md).
+
+Upgrading an existing installation takes three steps:
+
+1. Preparing the BHIMA software
+2. Preparing the database
+3. Running the upgrade
+
+Steps 1 and 2 can happen in parallel and can be done at a leasurely pace.  The third step is destructive and will commit the changes, deploying the new version at the location.
+
+These steps describe the typical deployment setup.  We assume that the BHIMA software is deployed in `$HOME/apps/`, `$VERSION` will be the next version name, and will refer to `$DATABASE` as the database name.
+
+## Step 1: Peparing the BHIMA software
+
+1. Go to https://github.com/ima-worldhealth/bhima/releases/latest in your web browser.
+2. Copy the link to the `$VERSION.tar.gz` file.
+2. Connect to the server with SSH
+3. Start a `screen` session in case the connection drops during the upgrade.
+4. Change directory into `$HOME/apps/`.
+5. Download the latest release with `wget -c "$URL"`.  This will create `$VERSION.tar.gz` in the `$HOME/apps/` directory.
```

Comment:
Where is URL defined?

Reply:
It isn't .  Unfortunately, GitHub doesn't let you know what the release URL is ahead of time.